doc/modules/preprocessing.rst

```diff
@@ -540,6 +540,15 @@ columns for this feature will be all zeros
     array([[1., 0., 0., 0., 0., 0.]])
 
 
+Missing categorical features in the training data can be handled by specifying what happens to them using the ``handle_missing`` parameter. The values for this can be one of :
+
+`all-missing`: This will replace all missing rows with NaN.
```

Comment:
This kind of operational description belongs in the class docstring. Here you would focus on the benefits or use-cases of one or another approach.